src/platform/Tizen/ConnectivityManagerImpl.cpp

```diff
@@ -224,6 +253,56 @@ bool ConnectivityManagerImpl::IsWiFiManagementStarted()
     return isActivated;
 }
 
+CHIP_ERROR ConnectivityManagerImpl::GetWiFiBssId(ByteSpan & value)
+{
+    uint8_t bssId[kMaxHardwareAddrSize];
+    CHIP_ERROR err = Internal::WiFiMgr().GetBssId(bssId);
+    VerifyOrReturnError(err == CHIP_NO_ERROR, err);
+
+    value = ByteSpan(bssId, 6);
```

Comment:
This is handing a pointer to the stack of this function out to the caller.  That will not work right.

Reply:
I'm sorry but i didn't understand your intention exactly. which stack do you mean to hand the pointer to the caller?

Reply:
`ByteSpan(bssId, 6)` is handing out, via the outparam, `&bssId[0]`.  But when this function returns that memory will no longer be holding anything useful, and when the caller tries to read the data they will best-case get garbage and worst-case crash.

Reply:
If so, does it mean that `ByteSpan()` is likely to occur on other platforms (Linux, etc.) where it is used?

Reply:
Just looked at the Linux code, and it's buggy in the same way, yes....

Reply:
And so is the NXP code.

Reply:
It seems that most platforms are the same way...

Reply:
Yep, please file issues.  That's a security bug right there...

Reply:
The real problem is that the API signature is broken.  I strongly suggest leaving this un-implemented for now, filing a followup issue to implement, and filing an issue to change the signature to take MutableByteSpan....

Reply:
Could you check the source code I modified?
```cpp
    MutableByteSpan mutableByteSpan = MutableByteSpan(bssId, 6);
    err = CopySpanToMutableSpan(value, mutableByteSpan);
    if (err != CHIP_NO_ERROR)
    {
        ChipLogProgress(DeviceLayer, "Failed to copy span data");
        return err;
    }

    return CHIP_NO_ERROR;
```

Reply:
That's copying data the wrong direction, no?

The only way to use the current API is to allocate the memory for the bssid somewhere and keep it alive forever.  See above about "the API signature is broken".

Reply:
Yes, it is the wrong direction, mistake.
We can store it in an internal handle called WiFiMgr. so it can use data pointer and size stored in ByteSpan.
Please check the contents of the next patch.